wiki/Commands/potion/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: .potion Command
+layout: wiki
+category: Command
+---
+Adds, sets or removes effects of the held potion. Accepts arguments in groups of 3 for add and set modes. Can set multiple effects to one potion. Make sure it fits the chat bar.
+
+## Syntax
+
+- `remove <effect>`
+- `add (<effect> <amplifier> <duration>)...`
+- `set (<effect> <amplifier> <duration>)...`
+or longer:
```

Comment:
The "or longer" part doesn't make much sense anymore, as the syntax above it already covers the longer versions.